transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -221,12 +263,13 @@ export class SmallWebRTCTransport extends Transport {
 
     this.state = "connecting";
 
-    this._webrtcUrl =
-      connectParams?.webrtcUrl ??
-      connectParams?.connectionUrl ??
-      this._webrtcUrl;
-    if (!this._webrtcUrl) {
-      logger.error("No url provided for connection");
+    // Note: There is no need to validate the params here, as they were already
+    //       validated and fixed in the parent class's connect() method (which calls
+    //       _validateConnectionParams() and passes the result to _connect()).
+    this._webrtcRequest =
+      connectParams?.webrtcRequestParams ?? this._webrtcRequest;
```

Comment:
Just confirming: we don't need to invoke `_resolveRequestInfo` again here because we already did it inside `_validateConnectionParams`, right?